### ISSUE TITLE
Define cross-WP team for Distribution & Deployment

### ISSUE THREAD

**Issue by proycon** [CLARIAH/clariah-plus#39]

**Comment by ddeboer** [comment]
Is this about infrastructure, CI/CD, packaging etc.? In that case, sounds interesting!

**Comment by proycon** [comment]
Yes, the exact scope of this task is still a bit vaguer than the others at this point, but that's the gest of it. Things like tools to data should fit well under this umbrella.

Welcome aboard, I added you to the team ;)

**Comment by proycon** [comment]
To sketch some more context; I also see this group as the logical continuation of what used to be the DevOps IG, there's no reason for an extra IG DevOps anymore in my view. This group will focus more on actual work like specifications and implementations; basically in the same line as we already did with the software & infrastructure requirements.

**Comment by proycon** [comment]
@tvermaut I think you should definitely be in this team as the way I'm formulating it now it covers a lot of the WP2 core activities, you may even want to (co-)coordinate it.

**Comment by proycon** [comment]
@mmisworking I suppose you have more Team-CI-collegues that could be added to this team as they work on CLARIAH infrastructure?

**Comment by mmisworking** [comment]
Indeed, there are more people involved in this task . Just have to figure out what usernames they want to use in GiTHub

**Comment by jblom** [comment]
@proycon @mmisworking I see I am already part of the members, which is good :-)

I think a great (initial) exercise would be to go through the architecture and CI/CD cycle for the Media Suite (ansible + VMs) & DANE (in k8s) and see which parts fit the CLARIAH infra+software reqs and which parts could be (conveniently) deployed in the CLARIAH cluster.

So looking forward to that and the broad discussions here.

**Comment by JessedeDoes** [comment]
I would like to follow this up for INT fttb, I will try to get myself replaced by a software engineer later on.

**Comment by proycon** [comment]
@JessedeDoes Perfect! added you to the team
